cmd/kube_create.go

```diff
@@ -198,10 +198,15 @@ var kubeCreateCmd = &cobra.Command{
 			return err
 		}
 
-		kubernetesVersion, err := fetchKubernetesVersion()
+		kubernetesVersion, err := cmd.Flags().GetString("version")
```

Comment:
I think can be cool to check if asking version exist, example:
https://github.com/kubernetes/kubernetes/releases/tag/v1.13.4
https://github.com/kubernetes/kubernetes/releases/tag/v{version}
https://github.com/kubernetes/kubernetes/releases/tag/{version}

Reply:
https://github.com/exoscale/cli/blob/742b3ae2000994b95557234591e038f9b0639b23/cmd/kube_create.go#L518-L544

Reply:
Thanks @pierre-emmanuelJ & @greut, will do this.